include/windows/alloca.h

```diff
@@ -0,0 +1,4 @@
+
+#pragma once
```

Comment:
The commit message needs more detail.  What are you trying to build on windows that currently doesn't build?  Also, please change the commit title to something like:

```
core/windows: Add missing header files to build <xxx>
```
This makes it easier to locate where a commit is making changes when reading through git log.